Framework/Kernel/inc/MantidKernel/ITimeSeriesProperty.h

```diff
@@ -43,13 +43,11 @@ class ITimeSeriesProperty {
   virtual std::pair<double, double> averageAndStdDevInFilter(const std::vector<SplittingInterval> &filter) const = 0;
   /// Return the time series's times as a vector<DateAndTime>
   virtual std::vector<Types::Core::DateAndTime> timesAsVector() const = 0;
-  /// Returns the calculated time weighted average value
-  virtual double timeAverageValue() const = 0;
```

Comment:
you'll have to update the exposure of `timeAverageValue` to the python API in `Framework/PythonInterface/mantid/kernel/src/Exports/TimeSeriesProperty.cpp`

Reply:
I bet that is a couple of the broken tests right there. Thanks for the catch!